pkg/images/enricher/enricher_impl.go

```diff
@@ -259,6 +266,7 @@ func (e *enricherImpl) EnrichImage(ctx context.Context, enrichContext Enrichment
 	// here.
 	if err != nil {
 		errorList.AddError(err)
+		errorList.AddError(delegateErr)
```

Comment:
Any reason not using `AddErrors(err, delegateErr)`?

Reply:
Why mark this resolved?

Reply:
fixed